doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,281 @@
+| (any)    | `<USER_NAME>`        | -         | `true`         | Name of the user accessing the Bucket (requires  write permissions) |
+| (any)    | `<PASSWORD>`         | -         | `true`         | Password for accessing the Bucket (requires  write permissions)    |
+| (any)    | `<BACKEND>`          | yes       | -              | Which backend to use, must be either `onprem` or `saas`            |
+| `onprem` | `<HOST>`             | yes       | -              | Fully qualified Hostname or ip Address                             |
+| `onprem` | `<PORT>`             | -         | `2580`         | Port of the BucketFS Service                                       |
+| `onprem` | `<SERVICE_NAME>`     | yes       | `bfsdefault`   | Name of the BucketFS Service                                       |
+| `onprem` | `<BUCKET_NAME>`      | yes       | `default`      | Name of the Bucket                                                 |
+| `onprem` | `<PATH>`             | -         | (empty / root) | Path inside the Bucket                                             |
+| `onprem` | `<VERIFY>`           | -         | `true`         | Whether to apply TLS security to the connection                    |
+| `saas`   | `<SAAS_ACCOUNT_ID>`  | yes       | -              | Account ID for accessing an SaaS database instance                 |
+| `saas`   | `<SAAS_DATABASE_ID>` | yes       | -              | Database ID of an Exasol SaaS database instance                    |
+| `saas`   | `<SAAS_PAT>`         | yes       | -              | Personal access token for accessing an SaaS database instance      |
+
+### AAF Python Package
+
+The latest version of AAF can be obtained from [pypi](https://pypi.org), see also the [releases on GitHub](https://github.com/exasol/advanced-analytics-framework/releases):
+
+```bash
+pip install exasol-advanced-analytics-framework
+```
+
+### Script Language Container (SLC)
+
+Exasol executes User Defined Functions (UDFs) in an isolated Container called _SLC_.
+
+Running the AAF requires a custom SLC. The following command
+* downloads the specified version `<VERSION>` (preferrably the latest) of a prebuilt AAF SLC from the [AAF releases](https://github.com/exasol/advanced-analytics-framework/releases/latest) on GitHub,
+* uploads the file into the BucketFS,
+* and registers it to the database.
+
+```shell
+LANGUAGE_ALIAS=PYTHON3_AAF
+python -m exasol_advanced_analytics_framework.deploy language-container \
+    --dsn <DB_HOST:DB_PORT> \
+    --db-user <DB_USER> \
+    --db-pass <DB_PASSWORD> \
+    --bucketfs-name <BUCKETFS_NAME> \
+    --bucketfs-host <BUCKETFS_HOST> \
+    --bucketfs-port <BUCKETFS_PORT> \
+    --bucketfs-user <BUCKETFS_USER> \
+    --bucketfs-password <BUCKETFS_PASSWORD> \
+    --bucket <BUCKETFS_NAME> \
+    --path-in-bucket <PATH_IN_BUCKET> \
+    --version <VERSION> \
+    --language-alias "$LANGUAGE_ALIAS"
```

Comment:
Hm, using $ and <> looks weird, you probably want to show that the language alias is needed in two places

Reply:
Yes, I already added a back-reference in the next example commandline, But see next push for
* consistently only use `$` references
* add a short rationale why environment variable `LANGUAGE_ALIAS` is set before